x-pack/plugins/index_management/public/sections/index_list/components/index_actions_context_menu/index_actions_context_menu.js

```diff
@@ -475,7 +449,7 @@ class IndexActionsContextMenuUi extends Component {
       <div>
         {this.state.renderConfirmModal ? this.state.renderConfirmModal(this.closeConfirmModal, getHttpClient()) : null}
         <EuiPopover
-          id={`contextMenu${entity}`}
+          id={`contextMenu${selectedIndexCount === 1 ? 'index' : 'indices'}`}
```

Comment:
I don't see a compelling reason to make this dynamic so I'd err on the side of keeping the `id` static here. Let's just hardcode it as `contextMenuIndices`.